### ISSUE TITLE
First load of first Web part in a playlist leaves too much white space after the content 

### ISSUE THREAD

**Issue by peterjkrebs** [pnp/custom-learning-office-365#452]
>**PROVISIONING NOTE**: If you are having issues with the SharePoint look book while trying to provision M365 learning pathways please do not post that issue to this repo as we cannot help you here. Please post all questions and issues for the SharePoint look book to [SharePoint/sp-provisioning-service GitHub Issues List](https://github.com/SharePoint/sp-provisioning-service/issues).

>**WEB PART NOT LOADING**: If you are getting the error: "Microsoft 365 learning pathways has a configuration issue. Ask your administrator for assistance". Please review the [Diagnosing Pre-Check](https://github.com/pnp/custom-learning-office-365/blob/main/installation/DiagnosisPreCheck.m the d) guidance before submitting your bug.

## Describe the bug

Click on the First asset in the Playlist of Six Simple Steps.
Excessive whitespace appears below the content.
Click on the next asset. The whitepace behaves as expected.
Click back to the initial asset, the excessive white space is now removed.

### To Reproduce

>Steps to reproduce the behavior

See above

**Comment by juliemturner** [comment]
This is a duplicate of issue #171 -- We are waiting on the fix from support.office.com
